Pcap++/src/PcapFilter.cpp

```diff
@@ -63,8 +67,9 @@ bool BpfFilterWrapper::setFilter(const std::string& filter, LinkLayerType linkTy
 			return false;
 		}
 
+		// TODO: Do we technically need 'free_program' here? Both m_program and m_filter string are overwritten directly after that.
```

Comment:
I agree that we should remove this.

Reply:
Yes I agree also

Reply:
Removed the call in df9f37e. Would still like an opinion on the previous comment tho.

Reply:
current way is fine.